infrastructure/ansible/roles/traffic_ops/templates/postinstall.input.j2

```diff
@@ -92,19 +92,31 @@
       "hidden": "1"
     }
   ],
-  "/opt/traffic_ops/app/db/dbconf.yml": [
+  "/opt/traffic_ops/app/conf/production/tv.conf": [
```

Comment:
Nope, it turns out the answers to those questions were never actually used since the `dbconf.yml` file is generated via the answers for `database.conf`. So, I removed them.